### ISSUE TITLE
Zuul: URISyntaxException when setting route.url with trailing slash

### ISSUE THREAD

**Issue by msparer** [spring-cloud/spring-cloud-netflix#583]
Using spring-cloud-starter-zuul 1.0.3.RELEASE, the problem occurs if you define a route like this (using spring-boot in this example):

```
zuul:
  routes:
    foos:
      path: /foos/**
      url: http://localhost:8080/
```
The trailing slash of the ```url``` always gets prepended to the delegation request. E.g.

```curl -X GET http://zuulhost/foos/1```

results in a ```GET //1``` to ```url``` (instead of ```GET /1```). This particular example does work though, a call to root doesn't however:

```curl -X GET http://zuulhost/foo/``` --> ```URISyntaxException``` as the request is ```GET //```

The current workaround of course is to define ```url``` without trailing slash, then everything works as expected:

```
zuul:
  routes:
    foos:
      path: /foos/**
      url: http://localhost:8080
```

tl;dr:
```ZuulRoute``` should take care of trailing slashes in ```url``` on delegation
